mitosheet/src/components/taskpanes/CannotCreateComm/CannotCreateCommTaskpane.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { DOCUMENTATION_LINK_INSTALL } from "../../../data/documentationLinks";
+import { CommCreationErrorStatus } from "../../../jupyter/comm";
+import { UIState, UserProfile } from "../../../types"
+import GetSupportButton from "../../elements/GetSupportButton";
+import Spacer from "../../layout/Spacer";
+
+import DefaultTaskpane from "../DefaultTaskpane/DefaultTaskpane";
+import DefaultTaskpaneBody from "../DefaultTaskpane/DefaultTaskpaneBody";
+import DefaultTaskpaneHeader from "../DefaultTaskpane/DefaultTaskpaneHeader";
+
+
+interface CannotCreateCommTaskpaneProps {
+    setUIState: React.Dispatch<React.SetStateAction<UIState>>;
+    userProfile: UserProfile
+    commCreationErrorStatus: CommCreationErrorStatus
+}
+
+
+/* 
+    This is the CannotCreateComm taskpane, which is displayed when
+    Mito cannot hook up to the backend properly
+*/
+const CannotCreateCommTaskpane = (props: CannotCreateCommTaskpaneProps): JSX.Element => {
```

Comment:
Since all of our logging goes through the backend, we aren't going to be able to see when the comm cannot be created. If I remember correctly, we used to have logging that was sent from the javascript. Do you think we should bring this back so we can keep an eye on this very important infrastructure?

Reply:
We can do this next sprint!